libs/api_common/windows_platform_common.cpp

```diff
@@ -79,6 +79,19 @@ const EbpfProgramType windows_sock_addr_program_type = {
     (uint64_t)&EBPF_PROGRAM_TYPE_CGROUP_SOCK_ADDR,
     {"cgroup/connect4", "cgroup/connect6", "cgroup/recv_accept4", "cgroup/recv_accept6"}};
 
+//
+// SOCK_OPS.
+//
+const ebpf_context_descriptor_t g_sock_ops_context_descriptor = {
+    sizeof(bpf_sock_ops_t),
+    -1, // Offset into ctx struct for pointer to data, or -1 if none.
+    -1, // Offset into ctx struct for pointer to data, or -1 if none.
+    -1, // Offset into ctx struct for pointer to metadata, or -1 if none.
```

Comment:
this is also duplicated in net_ebpf_ext_program_info.h.  Is it easy to have it be in one place instead of duplicated in both places?  If not easy, then ok as is.

Reply:
There was a similar feedback for SOCK_ADDR program context descriptor as well. I think having a common header was problematic. Since this should be addressed by #223, I am leaving this as is.